src/main/kotlin/com/github/spotbugs/snom/internal/SpotBugsRunner.kt

```diff
@@ -32,115 +32,102 @@ abstract class SpotBugsRunner {
-            args.add(task.baselineFile.get().asFile.absolutePath)
+            add("-excludeBugs")
+            add(task.baselineFile.get().asFile.absolutePath)
         }
         if (task.onlyAnalyze.isPresent) {
-            args.add("-onlyAnalyze")
-            args.add(task.onlyAnalyze.get().stream().collect(Collectors.joining(",")))
+            add("-onlyAnalyze")
+            add(task.onlyAnalyze.get().joinToString(","))
         }
-        args.add("-projectName")
-        args.add(task.projectName.get())
-        args.add("-release")
-        args.add(task.release.get())
+        add("-projectName")
+        add(task.projectName.get())
+        add("-release")
+        add(task.release.get())
         val file = task.analyseClassFile.asFile.get()
         task.classes?.let { generateFile(it, file) }
-        args.add("-analyzeFromFile")
-        args.add(file.absolutePath)
-        args.addAll(task.extraArgs.getOrElse(emptyList()))
-        log.debug("Arguments for SpotBugs are generated: {}", args)
-        return args
+        add("-analyzeFromFile")
+        add(file.absolutePath)
+        addAll(task.extraArgs.getOrElse(emptyList()))
+        log.debug("Arguments for SpotBugs are generated: {}", this)
     }
 
     private fun createFileForAuxClasspath(task: SpotBugsTask): String {
-        val auxClasspath =
-            task.auxClassPaths.files.stream()
-                .map { obj: File -> obj.absolutePath }
-                .collect(Collectors.joining("\n"))
+        val auxClasspath = task.auxClassPaths.files.asSequence()
+            .map { obj: File -> obj.absolutePath }
+            .joinToString("\n")
+        val auxClasspathFile = task.auxclasspathFile.map {
+            it.asFile.toPath()
+        }.get()
         try {
-            val auxClasspathFile =
-                task.auxclasspathFile.map {
-                    it.asFile.toPath()
-                }.get()
-            try {
-                Files.createDirectories(auxClasspathFile.parent)
-                if (!Files.exists(auxClasspathFile)) {
-                    Files.createFile(auxClasspathFile)
-                }
-                Files.write(
-                    auxClasspathFile,
-                    auxClasspath.toByteArray(),
-                    StandardOpenOption.TRUNCATE_EXISTING,
-                )
-                return auxClasspathFile.normalize().toString()
-            } catch (e: Exception) {
-                throw GradleException(
-                    "Could not create auxiliary classpath file for SpotBugsTask at " +
-                        auxClasspathFile.normalize().toString(),
-                    e,
-                )
+            auxClasspathFile.parent.createDirectories()
+            if (!auxClasspathFile.exists()) {
+                auxClasspathFile.createFile()
             }
-        } catch (e: Exception) {
-            throw GradleException("Could not create auxiliary classpath file for SpotBugsTask", e)
```

Comment:
There are 2 cathing, but seems they are catching for the same `IOException`, might be a typo here? I've replaced with `catch (e: IOException)`.